README.md

```diff
@@ -69,6 +71,13 @@ To only run the integration tests:
 go test -run Integration
 ```
 
+### Documentation 
+
+The Amazon QLDB Go Driver adheres to GoDoc standards and the documentation can be found [here](https://pkg.go.dev/github.com/awslabs/amazon-qldb-driver-go/qldbdriver?tab=doc).
+
+You can generate the docstring HTML locally by running the following in the root directory of this repository:
+```godoc -http=:6060```
+
 ## Getting Help
 
 Please use these community resources for getting help.
```

Comment:
Missing the entry for the QLDB Forum that's in the Python version.

Reply:
Added